torch/testing/_internal/common_methods_invocations.py

```diff
@@ -800,6 +886,27 @@ def sample_inputs(self, device, dtype, requires_grad=False):
                    promotes_integers_to_float=True,
                    handles_complex_extremals=False,
                    test_complex_grad=False),
+    OpInfo('svd',
+           op=torch.svd,
+           dtypes=floating_and_complex_types(),
+           test_inplace_grad=False,
+           supports_tensor_out=False,
+           sample_inputs_func=sample_inputs_svd,
+           decorators=[skipCUDAIfNoMagma, skipCPUIfNoLapack],
+           skips=(
+               # gradgrad checks are slow
+               SkipInfo('TestGradients', 'test_fn_gradgrad', active_if=(not TEST_WITH_SLOW)),
+               # cuda gradchecks are very slow
+               # see discussion https://github.com/pytorch/pytorch/pull/47761#issuecomment-747316775
+               SkipInfo('TestGradients', 'test_fn_gradgrad', device_type='cuda'),
+               SkipInfo('TestGradients', 'test_fn_grad', device_type='cuda'))),
```

Comment:
I think we can keep the grad checks going, let's just exclude the gradgrad checks on CUDA for now

Reply:
Should CUDA gradgrad check be always skipped or tested only with `PYTORCH_TEST_WITH_SLOW`?

Reply:
Let's just unconditionally skip them on svd for now. @albanD and I are going to look at our test matrix more closely in January.